ethexe/processor/src/handling/run.rs

```diff
@@ -106,14 +106,32 @@ async fn run_in_async(
             }
 
             for (program_id, journal) in super_journal {
-                let mut handler = JournalHandler {
-                    program_id,
-                    controller: TransitionController {
-                        transitions: in_block_transitions,
-                        storage: &db,
-                    },
+                let controller = TransitionController {
```

Comment:
handled message's origin may be thrown through receiver with journal

Reply:
Fixed